web/profiles/custom/yalesites_profile/modules/custom/ys_views_basic/src/ViewsBasicManager.php

```diff
@@ -122,11 +142,20 @@ public function getViewModeLabel($type, $view_mode) {
     return self::ALLOWED_ENTITIES[$type]['view_modes'][$view_mode];
   }
 
+  /**
+   * Returns a tag label given a term ID.
+   */
+  public function getTagLabel($tag) {
```

Comment:
I didn't think of this before, but would you mind typehinting functions? For example:
```
/**
  * Returns a tag label given a term ID.
  * 
  * @param int $tag
  *   The ID of a taxonomy term.
  * 
  * @return string
  *   The label of the given taxonomy term or an empty string.
  */
  public function getTagLabel(int $tag): string {
```

Reply:
Yup, thanks for the reminder. Added for all functions.